modules/local/mirdeep2.nf

```diff
@@ -0,0 +1,119 @@
+// Import generic module functions
```

Comment:
I think it would be nice to stick with the structure of one module per file and then combin the modules into subworkflows